app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileRenameActivityTest.kt

```diff
@@ -118,6 +126,21 @@ class ProfileRenameActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testProfileRenameActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
```

Comment:
This test is failing in CI check, please fix this.

Reply:
@pratyusu Why is the test failing ? Can you please point me out ? And also, is there a way to run the CI tests locally ?

Reply:
Yes, please take a look [here](https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing#running-robolectric-tests).

Reply:
Dome

Reply:
In this you might need to pass `ApplicationContext` instead of `context` to make it work.